src/axes.js

```diff
@@ -55,20 +55,8 @@ export function autoScaleLabels(channels, scales, {x, y, fx, fy}, dimensions, op
       fy.labelOffset = fy.axis === "left" ? facetMarginLeft : facetMarginRight;
     }
   }
-  if (x) {
-    autoAxisLabelsX(x, scales.x, channels.get("x"));
-    if (x.labelOffset === undefined) {
-      const {marginTop, marginBottom, facetMarginTop, facetMarginBottom} = dimensions;
-      x.labelOffset = x.axis === "top" ? marginTop - facetMarginTop : marginBottom - facetMarginBottom;
```

Comment:
By moving this to src/axis.js, we effectively lose control over this option for the *both* axis.